### ISSUE TITLE
Fix keyboard commands

### ISSUE THREAD

**Issue by maddyjoe** [RaspberryPiFoundation/editor-ui#302]
Examples of where keyboard commands/tabbing to navigate not working 
* Phil has mentioned that when he was navigating using a keyboard that "create new file" and the typing into the modal field and pressing enter to accept/save changes wasn't working
* settings --> you can open the settings menu but can't actually select light/dark mode or text size
* Possibly more thought needed around the sequence of tabbing through a page e.g. switching between tabs in the editor window, closing tabs, entering the editor for a file etc.
* Tabbing out of the editor pane is possible but non-obvious due to dual purpose of the tab key to navigate page and indent code

This ticket is to do a whizz through of key interactions with navs/menus to check they are working with keyboard for accessibility reasons. You can also tab out of it aka not a keyboard trap but should be.

**Comment by maddyjoe** [comment]
Closing as I've written this up as part of [a wider user story](https://github.com/RaspberryPiFoundation/digital-editor/issues/58) that covers a few other accessibility audit bugs/fixes
